extensions/shared/wrap-paid-block/style.scss

```diff
@@ -1,8 +1,10 @@
-.jetpack-paid-block__wrapper {
-	border: 1px solid var( --color-border-subtle );
-}
+@import '../styles/gutenberg-colors.scss';
+@import '../styles/gutenberg-variables.scss';
 
-.jetpack-paid-block__disabled {
-	opacity: 0.5;
-	padding: 10px;
+.block-editor-block-list__block:not( .is-selected ) .jetpack-paid-block__wrapper {
```

Comment:
Instead of referencing core class-names (which can be fragile), you could pass `.is-selected` class for `UpgradeNudge` in the HOC since you can see it in `props.isSelected`. Not a blocker and up to you how you see best.

Reply:
:+1: 5f13221